chaincode-labs/chaincode-podcast/2020-03-12-Matt-Corallo-and-Compact-Blocks-FIBRE-Episode-6.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Matt Corallo and Compact Blocks/FIBRE - Episode 6
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: []
+---
+
+Chaincode Labs podcast: Matt Corallo and Compact Blocks/FIBRE - Episode 6
```

Comment:
for comparison: https://btctranscripts.com/chaincode-labs/chaincode-podcast/2020-03-12-matt-corallo-compact-blocks-fibre/